Snippets/Snippets_3/Logging/UsingLogging.cs

```diff
@@ -8,6 +8,6 @@ public void SomeMethod()
         //your code
         Logger.Debug("Something interesting happened.");
     }
-    static ILog Logger = LogManager.GetLogger("Name");
+    readonly static ILog Logger = LogManager.GetLogger(typeof(ClassUsingLogging)); // Can also use a string
```

Comment:
Read-only makes sure that the the logger is jitted in the method or is todays .net compiler smart enough to see that it is only assigned in the static contructor? I also like its intent otherwise static would be noisy too right?